src/app/workspace/resource/resource-properties/resource-properties.component.ts

```diff
@@ -0,0 +1,305 @@
+    templateUrl: './resource-properties.component.html',
+    styleUrls: ['./resource-properties.component.scss']
+})
+export class ResourcePropertiesComponent implements OnInit, OnDestroy {
+
+    @ViewChild('displayEdit') displayEditComponent: DisplayEditComponent;
+    @ViewChild('addValue') addValueComponent: AddValueComponent;
+    /**
+     * parent resource
+     *
+     * @param (parentResource)
+     */
+    @Input() parentResource: ReadResource;
+
+    /**
+     * array of property object with ontology class prop def, list of properties and corresponding values
+     *
+     * @param (propArray)
+     */
+    @Input() propArray: PropertyInfoValues[];
+
+    /**
+     * array of system property object with list of system properties
+     *
+     * @param (systemPropArray)
+     */
+    @Input() systemPropArray: SystemPropertyDefinition[];
+
+    /**
+     * show all properties, even if they don't have a value.
+     *
+     * @param  (showAllProps)
+     */
+    @Input() showAllProps = false;
+
+    @Output() referredResourceClicked: EventEmitter<ReadLinkValue> = new EventEmitter<ReadLinkValue>();
+
+    @Output() referredResourceHovered: EventEmitter<ReadLinkValue> = new EventEmitter<ReadLinkValue>();
+
+    addButtonIsVisible: boolean; // used to toggle add value button
+    addValueFormIsVisible: boolean; // used to toggle add value form field
+    propID: string; // used in template to show only the add value form of the corresponding value
+
+    valueOperationEventSubscription: Subscription; // [] = []; // array of ValueOperationEvent subscriptions
+
+    constructor(
+        @Inject(DspApiConnectionToken) private _dspApiConnection: KnoraApiConnection,
+        private _valueOperationEventService: ValueOperationEventService,
+        private _valueService: ValueService
+    ) { }
+
+    ngOnInit() {
+        if (this.parentResource) {
+            // get user permissions
+            const allPermissions = PermissionUtil.allUserPermissions(
+                this.parentResource.userHasPermission as 'RV' | 'V' | 'M' | 'D' | 'CR'
+            );
+
+            // if user has modify permissions, set addButtonIsVisible to true so the user see's the add button
+            this.addButtonIsVisible = allPermissions.indexOf(PermissionUtil.Permissions.M) !== -1;
+        }
+
+        // listen for the AddValue event to be emitted and call hideAddValueForm()
+        // this.valueOperationEventSubscription = this._valueOperationEventService.on(Events.ValueAdded, () => this.hideAddValueForm());
+
+        // subscribe to the ValueOperationEventService and listen for an event to be emitted
+        this.valueOperationEventSubscription = this._valueOperationEventService.on(
+            Events.ValueAdded, (newValue: AddedEventValue) => {
+                this.addValueToResource(newValue.addedValue);
+                this.hideAddValueForm();
+            });
+
+        this.valueOperationEventSubscription = this._valueOperationEventService.on(
```

Comment:
I think you are reassigning the `Subscription` object here as opposed to adding an additional one. I would probably be better to have an array of subscriptions and then add each subscriptions like here https://github.com/dasch-swiss/dsp-ui-lib/blob/639ee44ce8d9bf4e2f259922c812401bbd7ad8aa/projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.ts#L123

Reply:
Yes, I had it as an array first, but then all the tests failed. In this case I have to spend more time (which I don't have) to update the tests. Hm...

Reply:
hmm strange, wouldn't it be similar to the tests here? https://github.com/dasch-swiss/dsp-ui-lib/blob/639ee44ce8d9bf4e2f259922c812401bbd7ad8aa/projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.spec.ts#L73

Reply:
I think I messed up because I had to copy code from ui-lib as well from the beol app. But we had a solution in ui-lib with `Subscription` in `property-view` and a similar thing with `Subscription[]` in `resource-view`. In this PR I had to bring the different logics (property-view and resource-view) into one component (resource-properties).

Reply:
But you solved it in 7e78424, right?